src/ts/components/dashboard/AccountMenu.ts

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
do we still want to keep this?

Reply:
we will use it later for tx list development for dashboard.

Reply:
@mytechtip is working on it. You may want to double check with him incase double work.